.coveragerc

```diff
@@ -251,6 +251,9 @@ omit =
     homeassistant/components/neato.py
     homeassistant/components/*/neato.py
 
+    homeassistant/components/ness_alarm.py
```

Comment:
Do we have enough coverage, 95%+, to remove this?

Reply:
Yep looks like we'd be good to remove it 👍 

![image](https://user-images.githubusercontent.com/1289759/50570862-0b555f80-0def-11e9-9c8c-fa79316cffda.png)